styles/main.css

```diff
@@ -122,30 +123,37 @@ nav ul li{
 }
 
 nav {
-    padding: 8px 2em;
-    border: 0 0 1px 0 dotted rgb(0,0,0,0.3);
-    margin: -4px 0 0 0 
+    border-bottom: 1px dotted rgb(0,0,0,0.3);

```

Comment:
The border property works a bit differently than the padding and margin properties with respect to setting the top/left/bottom/right values.  For border, you have to set them explicitly using border-top, etc.  (But you can still set the border-width, border-style, and border-color properties in one shot using border-top, etc, as you see in my correction here.)